nextflow_schema.json

```diff
@@ -864,6 +859,99 @@
             },
             "fa_icon": "fas fa-layer-group"
         },
+        "gprofiler2": {
+            "title": "gprofiler2",
+            "type": "object",
+            "description": "",
+            "default": "",
+            "properties": {
+                "gprofiler2_run": {
+                    "type": "boolean",
+                    "description": "Set to run gprofiler2 and do a pathway enrichment analysis.",
+                    "fa_icon": "fas fa-running"
+                },
+                "gprofiler2_organism": {
```

Comment:
OK for now, but when we sort out references I think we may need an optional generic species label at the top level of the report.

Reply:
Yep; ATM only gprofiler uses that param, that's why I made it specific, but can this change in the future